### PR TITLE
doc updates for graduating Windows + containerd support to stable

### PR DESCRIPTION
**Pull request by marosset**
Doc updates for graduating Windows + containerd support to stable

### CODE REVIEW

**Review by irvifa — COMMENTED** [review]
/sig windows

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (jsturtevant) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:48]: Should we start to drop off the 1.14/1.15 releases from this list to avoid confusion on what k8s versions are supported?

Comment (michmike) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:48]: agreed., we should also remove 1.16, right? since this will go out with 1.20 release

Comment (marosset) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:48]: Sounds good.

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:168]:
```suggestion
#### Run `kubeadm` to join the node
```

We usually format commands as monospace.

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:195]: nit:
```suggestion
If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, specify the name with `-netAdapterName`.
```

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:211]:
```suggestion
#### Run `kubeadm` to join the node
```

**Review by sftim — COMMENTED** [review]
I've suggested a few tweaks.

Overall this looks ready for tech review. @marosset can you recommend / nominate a reviewer?

**Comment by neolit123** [comment]
LGTM, thanks @marosset !

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (marosset) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:128]: Need to confirm with @immuzz that GMSA support for Windows Server 2019 will be available in Nov 2020 (and if not update this section)

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (marosset) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:249]: I think the Windows OS fixes for correct single file mount on Windows are widely available. 
Let me confirm and i'll update the docs here.

**Comment by marosset** [comment]
Thanks all, I'll try and address the comments today or tomorrow (I'm going to prioritize k/k PRs since code-freeze is this week).

/assign @michmike for technical review
@immuzz @craiglpeters  feel free to comment as well :)

@sturtevant / @benmoss to review adding-windows-nodes.md updates / instructions

**Comment by marosset** [comment]
@michmike I updated some of the details on the pause image in the intro-windows-in-kubernetes.md to point make it easier to
- find the current pause image to use
- point to new build process

PTAL

**Comment by raghvenders** [comment]
@marosset  - Is this a feature to be considered include in feature-gates.md?

**Comment by marosset** [comment]
@irvifa - I can squash commit before this gets merged.

@raghvenders - There is not a feature-gate associated with this enhancement / work.

**Comment by raghvenders** [comment]
@marosset  - as i see in the OS version table kubernetes version - "Kubernetes v1.14" is removed.

But i could see "https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#docker-ee" had - "FEATURE STATE: Kubernetes v1.14 [stable]"

Please do correct me you are not touching it as it is not containerd.

**Comment by marosset** [comment]
@raghvenders - I removed K8s versions 1.14-1.16 based on a previous comment in this PR that those versions are out of the servicing lifecycle for the project and that was done as general cleanup / is not directly related to containerd.

I can split those changes out into a separate PR (or seperate commits in this PR) if that would be helpful. 
I made the changes in same PR to avoid merge-conflicts since both PRs would be updating doc sections very close to each other.

**Comment by jimangel** [comment]
Can we get confirmation on https://github.com/kubernetes/website/pull/24862/files#r520826250 ? Otherwise:
/lgtm

**Comment by annajung** [comment]
/hold cancel
code merged

hi @michmike @immuzz, can you please provide tech review / lgtm?

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (michmike) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:127]: @marosset  were you able to clarify this?

**Review by michmike — APPROVED** [review]
/approve
/lgtm

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (sftim) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:60]: That's not a hyperlink @raghvenders - seems fine as it is.

**Comment by sftim** [comment]
@marosset would you be willing to squash these commits down? This repo preserves LGTM through squashes.

No worries if you are busy right now, but it would be nice to have that.

**Comment by sftim** [comment]
Otherwise, we can approve in a day or two anyway.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-24862--kubernetes-io-vnext-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/#joining-a-windows-worker-node

**Comment by marosset** [comment]
I was finally able to get the answers to an open question I had a a PR comment about GMSA support for Windows Server 2019.
Let me address that and a few other minor PR comments this AM. I'll squash changes down when I make those updates.

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (marosset) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:127]: Yes, looks like it will be available early 2021. Let me reopen that issue and update this caution section.

Comment (sftim) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:127]: Bear in mind that our default setting is to [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) entirely. It's OK _not_ to make a date commitment.

**Comment by sftim** [comment]
Ah, this'll need a rebase anyway.

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (marosset) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:248]: The Windows OS fixes are in place but we need to make an update in K8s which we'll do in v1.21

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (marosset) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:60]: I'll leave it as is

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
Comment (marosset) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:603]: There is information on how to set this in the kubelet in other parts of this document that were not updated here.
Normally users shouldn't need ever actually perform a pull for this image.

**Comment by marosset** [comment]
Let rebase on top of dev-1.20 again

**Comment by annajung** [comment]
lgtm from both tech and docs still stands,

/approve

**Pull request merged**
